DigitalLearningSolutions.Data.Tests/Services/SectionContentServiceTests.cs

```diff
@@ -316,5 +316,19 @@ public void Get_section_content_should_return_content_if_only_tutorial_status_is
             );
             result.Should().BeEquivalentTo(expectedSectionContent);
         }
+
+        [Test]
+        public void Get_section_content_post_learning_assessment_passed_should_convert_to_true()
```

Comment:
We should be able to test this when we use it in the view model? 🙂

Reply:
I thought it was sensible to have it here since the logic for it was in the data model. Do you think it would be better to have the test in the view model?

Reply:
But it's not to do with the service so doesn't belong in the service tests 🙂 We could have a separate test for the section content itself, or we could test the logic as part of the view model. As the logic is so small I don't really mind which one we go for. But either way we need more than one test to check that it's working. I think testing 0, 1 and > 1 passes should do. You could do that using `[TestCase]` 🙂

Reply:
Yep that's fine, I'll do it in the View Model test. Do you want me to add it as part of this PR or part of the integration PR as currently we aren't using any of these values in the view model?

Reply:
Fine to leave for now and do as part of the integration 🙂